Makefile

```diff
@@ -1,137 +1,205 @@
-PHP_CS_FIXER_FUTURE_MODE=1
-PHPSTAN=./phpstan.phar
-PHP-CS-FIXER=./php-cs-fixer-v2.phar
-PHPUNIT=vendor/bin/phpunit
-INFECTION=build/bin/infection.phar
+.DEFAULT_GOAL := help
```

Comment:
The new Makefile structure is as follows:

```Makefile
.DEFAULT_GOAL := help

.PHONY: help
help: ...

# Variables

...

# Commands

...

# Rules
```

PHONY targets are also declared as such right above the target: I believe we have already quite a few targets which makes it hard to follow hence having the PHONY target right above the declaration is both less error prone and easier to check IMO. IIRC it was a suggestion from @sanmai and I must say I never had to look back at it ever since I've adopted that practice